apps/minaintyg/src/pages/Certificates/components/CertificateList.test.tsx

```diff
@@ -0,0 +1,97 @@
+import { fakerFromSchema } from '@frontend/fake'
+import { render, screen, within } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import { certificateListItemSchema } from '../../../schema/certificateList.schema'
+import { CertificateList } from './CertificateList'
+
+const fakeCertificate = fakerFromSchema(certificateListItemSchema)
+
+it('Should sort certificates by year in descending order', () => {
+  const certificates = [
+    fakeCertificate({ timestamp: '2021-01-01' }),
+    fakeCertificate({ timestamp: '2022-01-01' }),
+    fakeCertificate({ timestamp: '2023-01-01' }),
+  ]
+
+  render(
+    <MemoryRouter>
+      <CertificateList certificates={certificates} />
+    </MemoryRouter>
+  )
+
+  const yearHeadings = screen.getAllByRole('heading', { level: 2 })
+  expect(yearHeadings).toHaveLength(3)
+  expect(yearHeadings[0]).toHaveTextContent('2023')
+  expect(yearHeadings[1]).toHaveTextContent('2022')
+  expect(yearHeadings[2]).toHaveTextContent('2021')
+})
+
+it('Should sort certificates by year in ascending order', () => {
+  const certificates = [
+    fakeCertificate({ timestamp: '2021-01-01' }),
+    fakeCertificate({ timestamp: '2022-01-01' }),
+    fakeCertificate({ timestamp: '2023-01-01' }),
+  ]
+
+  render(
+    <MemoryRouter>
+      <CertificateList certificates={certificates} order="ascending" />
+    </MemoryRouter>
+  )
+
+  const yearHeadings = screen.getAllByRole('heading', { level: 2 })
+  expect(yearHeadings).toHaveLength(3)
+  expect(yearHeadings[0]).toHaveTextContent('2021')
+  expect(yearHeadings[1]).toHaveTextContent('2022')
+  expect(yearHeadings[2]).toHaveTextContent('2023')
+})
+
+it('Should sort certificates in descending order per year', () => {
+  const certificates = [
```

Comment:
Since certificates is initialized the same the tests can be nested in a describe and certificates can be initialized in a beforeEach

Reply:
Every list of certificates are different.

Reply:
"Should sort certificates in descending order per year" and "Should sort certificates in ascending order per year" are the same